crypto/merkletrie/cache.go

```diff
@@ -145,8 +145,9 @@ func (mtc *merkleTrieCache) getNode(nid storedNodeIdentifier) (pnode *node, err
 	return
 }
 
-// prioritizeNode make sure to move the priorities of the pages according to
-// the accessed node identifier
+// prioritizePage make sure to adjust the priority of the given node id.
```

Comment:
`prioritizePage` in the comment and `prioritizeNode` in the code

Reply:
i will fix comment in a subsequent PR